log_init.py

```diff
@@ -127,17 +127,14 @@ def _add_log_level(level_name: str, level_num: int, method_name: Optional[str] =
     """
     Comprehensively adds a new logging level to the `logging` module and the
     currently configured logging class.
-
```

Comment:
I feel like these make sense to keep. They establish a break in thoughts in the docstring.

Reply:
Oh i agree, that probably was the black formatter removing it. I'll check on that